packages/components/pie-toast/src/toast.scss

```diff
@@ -1 +1,72 @@
 @use '@justeattakeaway/pie-css/scss' as p;
+
+.c-toast {
+  --toast-border-radius: var(--dt-radius-rounded-b);
+  --toast-background-color: var(--dt-color-container-inverse);
+  --toast-font-color: var(--dt-color-content-inverse);
+  --toast-font-size: #{p.font-size(--dt-font-body-s-size)};
+  --toast-line-height: #{p.line-height(--dt-font-body-s-line-height)};
+
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  min-height: 48px;
+  max-height: 122px;
+  min-width: 300px;
+  max-width: 420px;
+  
+  padding: var(--dt-spacing-a) var(--dt-spacing-c) var(--dt-spacing-a) var(--dt-spacing-d);
+  border-radius: var(--toast-border-radius);
+  gap: var(--dt-spacing-a);
+  background-color: var(--toast-background-color);
+  box-shadow: var(--dt-elevation-03);
+  color: var(--toast-font-color);
+  font-size: var(--toast-font-size);
+  line-height: var(--toast-line-height);
+
+  &-content-area{
+    display: flex;
+    gap: var(--dt-spacing-b);
+    justify-content: space-between;
+    
+  }
+
+  &-message-area {
+    display: flex;
+    align-items: center;
+    gap: var(--dt-spacing-b);
+    padding: calc(var(--dt-spacing-03) / 2) 0;
+
+    span {
+      text-overflow: ellipsis;
+      white-space: nowrap;
+      overflow: hidden;
+    }
+
+    &[isMultiline] {
```

Comment:
Heads-up about using custom attributes for styling – I'm just working on moving our custom attributes over to using classes instead (as the custom attributes aren't valid HTML, so we want to move this over so it's valid).

I should have a WIP done for this today that I can share, so I'll send it over as you can likely build this straight into your PR then rather than refactor later.

Reply:
@ashleynolan I would like to learn more about it.

Reply:
I think we could leverage data attributes `data-is-multiline` in order to stick with valid html. I'd love to discuss this a little more with you and the team.

Reply:
Heya – I've put a [draft PR up](https://github.com/justeattakeaway/pie/pull/1587/files) that shows switching to using classmap rather than data attributes. It's a method Lit recommend using (and it's super easy to use), so take a look at that and if you have any q's let me know.

I'm going to be working on moving all our components over to using `classMap` to make them valid – adding `data-` makes the styling look a bit cumbersome, as we tried it in a couple of our components and it makes the CSS look 🤮 😆